src/FITSIO.jl

```diff
@@ -147,6 +147,23 @@ include("header.jl")  # FITSHeader methods
 include("image.jl")  # ImageHDU methods
 include("table.jl")  # TableHDU & ASCIITableHDU methods
 
+"""
+    libcfitsio_version() -> VersionNumber
+
+### Purpose ###
+
+Return the version of the underlying CFITSIO library
+
+### Example ###
+
+```julia
+julia> using FITSIO
```

Comment:
Yes, it's necessary, unless documentation is configured to automatically run `using FITSIO` before each snippet.  See https://github.com/JuliaAstro/AstroLib.jl/pull/33#issuecomment-313680563 for the same issue in AstroLib.